frontend/src/scenes/insights/insightLogic.tsx

```diff
@@ -438,6 +439,12 @@ export const insightLogic = kea<insightLogicType>({
         loadedFilters: [(s) => [s.insight], (insight) => insight.filters],
         insightProps: [() => [(_, props) => props], (props): InsightLogicProps => props],
         insightName: [(s) => [s.insight], (insight) => insight.name],
+        canEditInsight: [
+            (s) => [s.insight],
+            (insight) =>
+                insight.effective_privilege_level == undefined ||
```

Comment:
The absence of something allowing edit permissions makes me wonder whether the backend trusts an API call or double-checks the person really does have edit permissions, and whether someone could break that accidentally... 

So non-blocking, but I wonder if we can integrate SAST or DAST security scanning into CI

Reply:
What do you mean by "absence of something allowing edit permissions", I'm not sure I got it?

Reply:
You can edit if effective_privilege_level is undefined.

And as a I re-read it, it's the insight that's set on... I parsed it as being on the person because I'd started wondering about how an attacker might try and alter an API call to see what happened.

So, ignore me :)